docs/modules/ROOT/pages/how-tos/synthesize.adoc

```diff
@@ -0,0 +1,256 @@
+
+# Configure restic password for cluster backups
+vault kv put -cas=0 clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cluster-backup password=$(pwgen -s 32 1)
+----
+<1> You can find the tenant ID on {portal-lieutenant-tenants}
+<2> The cluster ID which you chose in step 1a
+<3> Your usual VSHN LDAP login, generally `firstname.lastname`
++
+Check tenant-specific documentation for further tokens to configure in Vault.
++
+[NOTE]
+====
+For K8up,  S3 compatible storage is required.
+If the infrastructure does not provide any S3 compatible storage, you need to find an alternative.
+There are several options:
+
+* You can consume S3 storage from a different provider than the one hosting the cluster.
+* You can setup Minio inside the cluster to present an S3-compatible interface backed by a Kubernetes persistent volume.
+There's a https://github.com/projectsyn/component-minio/[Commodore component] for installing Minio.
+====
+
+. Install Steward on the cluster with the install URL displayed in the Portal
++
+CAUTION: Make sure the `kubectl` command in this step is executed against the cluster to synthesize
++
+[source,bash]
+----
+export KUBECONFIG=/path/to/new/cluster.kubeconfig
+kubectl create -f "${INSTALL_URL}"
+----
++
+[NOTE]
+====
+The install URL is only valid once and is only valid for 30 minutes after creating the cluster in Lieutenant.
+You can check the expiration date of the install URL by inspecting the cluster object on the {synfra-cluster}.
+If you don't see a field `.status.bootstrapToken.tokenValid` the token has either been used or has expired.
+
+[source,bash]
+----
+export CLUSTER_ID=<cluster id of cluster to synthesize>
+export KUBECONFIG=/path/to/synfra.kubeconfig
+
+export LIEUTENANT_NS=lieutenant-prod
+
+kubectl -n ${LIEUTENANT_NS} describe cluster ${CLUSTER_ID}
+----
+
+Use the following command to reset the cluster's bootstrap token
+
+[source,bash]
+----
+export CLUSTER_ID=<cluster id of cluster to synthesize>
+export KUBECONFIG=/path/to/synfra.kubeconfig
+
+export LIEUTENANT_NS=lieutenant-prod
+export LIEUTENANT_TOKEN=$(kubectl config view -o jsonpath='{.users[?(@.name == "syn-synfra")].user.token}'  --raw)
+export LIEUTENANT_AUTH="Authorization: Bearer ${LIEUTENANT_TOKEN}"
+
+curl -H "${LIEUTENANT_AUTH}" -H "Content-Type: application/json-patch+json" -X PATCH \
+  -d '[{ "op": "remove", "path": "/status/bootstrapToken" }]' \
+  "https://rancher.vshn.net/k8s/clusters/c-c6j2w/apis/syn.tools/v1alpha1/namespaces/${LIEUTENANT_NS}/clusters/${CLUSTER_ID}/status"
+----
+====
++
+This `kubectl create` command will install Steward on the cluster in the namespace "syn" and Steward will:
+
+* Generate SSH key which is then being used as a deploy key to access the Git repositories managed by the GitRepo objects.
+This SSH key will be sent to the Lieutenant API once generated.
+* Retrieve Git repository URL from Lieutenant
+* Bootstrap Argo CD and generate the "root" Application definition to connect to the GitOps repository
+
++
+[TIP]
```

Comment:
Separate how-to?

Reply:
Probably a good idea, but currently there's no indication that the portal will provide an install URL after creating the cluster, you just get dropped back to the cluster list.From what I can tell, hitting F5 at that point will show the popup containing the `kubectl` command containing the install URL.

For that reason, I've made this command to fetch the install URL a part of the regular instructions for now.

Reply:
Uh, we should check that again. It is supposed that the Portal shows the install URL immediately after creating a cluster.